### DISCUSSION TITLE
Helm Chart - v1.11.1

### DISCUSSION THREAD

**Discussion by tomkerkhove** [Azure/api-management-self-hosted-gateway#314] [Announcements]
[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/azure-api-management)](https://artifacthub.io/packages/helm/azure-api-management/azure-api-management-gateway/1.11.1)

*Chart: v1.11.1 | App: v2.7.2*

### Features

- Use container image v2.7.2 with vulnerability fixes ([release notes](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/Container-v2.7.2))

### Fixes / Changes

None.

### Breaking Changes

None.

### Removal

None.

**Full Changelog**: https://github.com/Azure/api-management-self-hosted-gateway/compare/v1.11.0...v1.11.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/v1.11.1'>Helm Chart - v1.11.1</a>.</em>

**Comment by TimMnz09** [comment]
@tomkerkhove The new version is not available on the artifact hub. 
https://artifacthub.io/packages/helm/azure-api-management/azure-api-management-gateway/1.11.1

**Reply by tomkerkhove** [reply]
Thanks for raising, I messed up the manifest but is fixed now and chart is reported on Artifact Hub
